star_tides/services/mongo/versions/20210404145400_description.py

```diff
@@ -0,0 +1,12 @@
+''' Mongodb migration. May not be used.
+'''
+# pragma pylint: disable=invalid-name
+from mongodb_migrations.base import BaseMigration
+
+
+class Migration(BaseMigration):
+    def upgrade(self):
+        self.db.test_collection.save({"new_column": "value"})
```

Comment:
Is this correct to have here?

Reply:
We will likely want migrations. This is the base for it, but I haven't quite gotten it to work yet.

Reply:
Can you add migrations as something in the future? Do we need it now?

Reply:
Deleted it.